src/libraries/System.ComponentModel.TypeConverter/tests/TrimmingTests/System.ComponentModel.TypeConverter.TrimmingTests.proj

```diff
@@ -1,12 +1,6 @@
 <Project DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
 
-  <ItemGroup>
```

Comment:
Can you leave these in place? The issue with the current behavior is that as soon as someone adds one `TestConsoleAppSourceFiles` item, the implicitly defined ones go away and the tests don't run.

I wish we had one model in these tests. 

cc @joperezr

Reply:
Right, I remember that. That's why I deleted all the `TestConsoleAppSourceFiles Include=...` lines. That would mean every file gets included and tested right?

Reply:
If you remove all of the `TestConsoleAppSourceFiles` items, then the infra searches for all `.cs` files in the current directory. As soon as you add 1 `TestConsoleAppSourceFiles` item, it stops doing that. I want us to stop relying on the implicit behavior, so people don't make mistakes in the future.

Reply:
Ah got it. Fixed now

Reply:
While @eerhardt point is correct in that the model is not ideal so one inclusion overrides the default, I think you are fine to keep it the way you had it (with them removed) since it would effectively be the same thing here.

Reply:
>If you remove all of the TestConsoleAppSourceFiles items, then the infra searches for all .cs files in the current directory. As soon as you add 1 TestConsoleAppSourceFiles item, it stops doing that. I want us to stop relying on the implicit behavior, so people don't make mistakes in the future.

I see, We could easily just remove the automatic globbing if you think that would be better. I'm happy to take care of that and find all Trimming tests depending on it and moving them to the manual specification instead.

Reply:
The other option would be to make it work like all other globbing behavior in projects (for example, the `*.cs` Compile globbing). If I don't want a `.cs` file to be a top level test, I write

`<TestConsoleAppSourceFiles Remove="SharedHelperFile.cs" />`. 

And if I need to set some metadata items for a specific file I write:

`<TestConsoleAppSourceFiles Update="MyTest.cs" SomeSwitch="true" />`. 

And then by default the globbing just works like people normally expect - it is always there. And adding/updating/removing the `TestConsoleAppSourceFiles` item doesn't affect other tests.

Reply:
I'd need to check if the Update part will work in this case since globbing will set the itemspec as the full path and I'm not sure if just filename and extension would match and appropriately set the switch, but I'll take a look to see if this could be done.